testing_and_setup/compass/ocean/global_ocean/cull_mesh.py

```diff
@@ -184,8 +185,12 @@ def removeFile(fileName):
     # Run command is:
     # ./MpasCellCuller.x  base_mesh.nc culled_mesh_preliminary.nc -m land_mask_final.nc
     # -p critical_passages_mask.nc
-    args = ['./MpasCellCuller.x', 'base_mesh.nc', 'culled_mesh_preliminary.nc',
-            '-m', 'land_mask_final.nc', '-p', 'critical_passages_mask.nc']
+    if options.preserve_floodplain:
```

Comment:
Perfect way to handle this more clearly and avoid accidental use by other scripts.

Reply:
I think it should work nicely. We just have to remember to use the flag in any coastal floodplain meshes.